examples/ligand_in_water/ligand_in_water.ipynb

```diff
@@ -0,0 +1,279 @@
+    "There are a few ways to convert the information in this trajectory to an Openff [`Topology`](https://docs.openforcefield.org/projects/toolkit/en/stable/api/generated/openff.toolkit.topology.Topology.html#openff.toolkit.topology.Topology) object. In this case, since we already know how many of which molecules we want, we'll use [`Topology.from_molecules`](https://docs.openforcefield.org/projects/toolkit/en/stable/api/generated/openff.toolkit.topology.Topology.html#openff.toolkit.topology.Topology.from_molecules), which takes a list of `Molecule` objects and assembles them into a `Topology`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1ecfae59-501e-41fd-b366-88a6975ac066",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "topology = Topology.from_molecules([ligand, *1000 * [water]])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "47e4ae9e-829a-4a4e-8ac1-93e698598fbd",
+   "metadata": {},
+   "source": [
+    "Next we'll assign the topology's box vectors using the same information on the MDTraj object:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0e6be87f-48e7-4023-bd46-fd09659c8559",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trajectory.unitcell_vectors * unit.nanometer\n",
+    "vectors = [\n",
+    "    trajectory.unitcell_vectors[0][0],\n",
+    "    trajectory.unitcell_vectors[0][1],\n",
+    "    trajectory.unitcell_vectors[0][2],\n",
+    "] * unit.nanometer\n",
+    "topology.box_vectors = vectors"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64b7ea41-b4c9-4ae0-95dd-416a7fb0299f",
+   "metadata": {},
+   "source": [
+    "The [\"Sage\"](https://openforcefield.org/community/news/general/sage2.0.0-release/) force field line (version 2.x.x) includes TIP3P  parameters for water, so we don't need to use multiple force fields to parametrize this topology.\n",
+    "\n",
+    "Note that the \"Parsley\" (version 1.x.x) line did *not* include TIP3P parameters, so loading in an extra force field was required."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9687547b-d6d0-4c38-b963-cca91b3127f7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sage = ForceField(\"openff-2.0.0.offxml\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "da7781fc-1adb-4ba9-af97-7c64649f36ef",
+   "metadata": {},
+   "source": [
+    "From here, we can create an ``Interchange`` object and promptly export it to an [``openmm.System``](http://docs.openmm.org/latest/api-python/generated/openmm.openmm.System.html#openmm.openmm.System):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "54b39faf-eae9-4550-92bc-68794432ec59",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if False:\n",
+    "    interchange = Interchange.from_smirnoff(force_field=sage, topology=topology)\n",
+    "    system = interchange.to_openmm(combine_nonbonded_forces=True)\n",
```

Comment:
Something is broken in `MDConfig` parsing constraints, whether I use the constrained or unconstrained version of Sage it chokes on something.